package.json

```diff
@@ -16,9 +16,11 @@
     "prebuildify": "prebuildify -t 10.16.0 --napi --strip --postinstall \"node scripts/postinstall.js --prebuild\"",
     "download-prebuilds": "prebuildify-ci download",
     "submodule": "git submodule update --recursive --init",
-    "test": "standard && nyc node test/index.js",
+    "test": "npm run check-formatting && nyc node test/index.js",
     "test-integration": "node test/integration.js",
-    "hallmark": "hallmark --fix"
+    "hallmark": "hallmark --fix",
+    "check-formatting": "prettier --check \"lib/**/*.{ts,tsx}\"",
+    "fix-formatting": "prettier --write \"lib/**/*.{ts,tsx}\""
```

Comment:
I would propose rename "check-formatting" to "lint" and "fix-formatting" to "lint-fix". It's shorter and tab completion (in bash) works better.